### PR TITLE
Stop running ingress conformance tests from vendor/ and use our own copy.

### PR DESCRIPTION
**Pull request by tcnghia**

### CODE REVIEW

File: test/conformance/ingress/basic_test.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/basic_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/grpc_test.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/grpc_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/headers_test.go [hunk 3]
```diff
@@ -0,0 +1,185 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/headers_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/hosts_test.go [hunk 4]
```diff
@@ -0,0 +1,70 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/hosts_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/path_test.go [hunk 5]
```diff
@@ -0,0 +1,251 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/path_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/percentage_test.go [hunk 6]
```diff
@@ -0,0 +1,135 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/percentage_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/retry_test.go [hunk 7]
```diff
@@ -0,0 +1,140 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/retry_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/timeout_test.go [hunk 8]
```diff
@@ -0,0 +1,111 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/timeout_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/tls_test.go [hunk 9]
```diff
@@ -0,0 +1,76 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/tls_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/update_test.go [hunk 10]
```diff
@@ -0,0 +1,204 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/update_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/util.go [hunk 11]
```diff
@@ -0,0 +1,947 @@
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/util.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/websocket_test.go [hunk 12]
```diff
@@ -0,0 +1,203 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline test/conformance/ingress/websocket_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/basic_test.go (hunk 1 above)
Comment (mattmoor) [inline test/conformance/ingress/basic_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/grpc_test.go (hunk 2 above)
Comment (mattmoor) [inline test/conformance/ingress/grpc_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/headers_test.go (hunk 3 above)
Comment (mattmoor) [inline test/conformance/ingress/headers_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/hosts_test.go (hunk 4 above)
Comment (mattmoor) [inline test/conformance/ingress/hosts_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/path_test.go (hunk 5 above)
Comment (mattmoor) [inline test/conformance/ingress/path_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/percentage_test.go (hunk 6 above)
Comment (mattmoor) [inline test/conformance/ingress/percentage_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/retry_test.go (hunk 7 above)
Comment (mattmoor) [inline test/conformance/ingress/retry_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/timeout_test.go (hunk 8 above)
Comment (mattmoor) [inline test/conformance/ingress/timeout_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/tls_test.go (hunk 9 above)
Comment (mattmoor) [inline test/conformance/ingress/tls_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/update_test.go (hunk 10 above)
Comment (mattmoor) [inline test/conformance/ingress/update_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/util.go (hunk 11 above)
Comment (mattmoor) [inline test/conformance/ingress/util.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/websocket_test.go (hunk 12 above)
Comment (mattmoor) [inline test/conformance/ingress/websocket_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/basic_test.go (hunk 1 above)
Comment (mattmoor) [inline test/conformance/ingress/basic_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/grpc_test.go (hunk 2 above)
Comment (mattmoor) [inline test/conformance/ingress/grpc_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/headers_test.go (hunk 3 above)
Comment (mattmoor) [inline test/conformance/ingress/headers_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/hosts_test.go (hunk 4 above)
Comment (mattmoor) [inline test/conformance/ingress/hosts_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/path_test.go (hunk 5 above)
Comment (mattmoor) [inline test/conformance/ingress/path_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/percentage_test.go (hunk 6 above)
Comment (mattmoor) [inline test/conformance/ingress/percentage_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/retry_test.go (hunk 7 above)
Comment (mattmoor) [inline test/conformance/ingress/retry_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/timeout_test.go (hunk 8 above)
Comment (mattmoor) [inline test/conformance/ingress/timeout_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/tls_test.go (hunk 9 above)
Comment (mattmoor) [inline test/conformance/ingress/tls_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/update_test.go (hunk 10 above)
Comment (mattmoor) [inline test/conformance/ingress/update_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/util.go (hunk 11 above)
Comment (mattmoor) [inline test/conformance/ingress/util.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: test/conformance/ingress/websocket_test.go (hunk 12 above)
Comment (mattmoor) [inline test/conformance/ingress/websocket_test.go:4]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

**Comment by tcnghia** [comment]
/close
@dprotaso implemented a better approach

**Pull request closed without merging**